bindings/pyroot_experimental/PyROOT/src/PyROOTModule.cxx

```diff
@@ -74,6 +76,8 @@ static PyMethodDef gPyROOTMethods[] = {{(char *)"AddDirectoryWritePyz", (PyCFunc
                                         (char *)"Initialize interactive ROOT use from Python"},
                                        {(char *)"InstallGUIEventInputHook", (PyCFunction)PyROOT::RPyROOTApplication::InstallGUIEventInputHook, METH_NOARGS,
                                         (char *)"Install an input hook to process GUI events"},
+                                       {(char *)"_CPPInstance__expand__", (PyCFunction)PyROOT::CPPInstanceExpand, METH_VARARGS,
+                                        (char *)"Helper method for pickling"},
```

Comment:
Perhaps the description should say that this is the method that deserializes a pickled object?